### PR TITLE
Pages and Components added

### PR DESCRIPTION
**Pull request by Mahi24-prog**
### Pages and Components added 

- Pages and Components added in react template
- Created separate components

### CODE REVIEW

**Review by sakleprajwal — APPROVED** [review]
No changes required. Please go ahead

File: src/CSS/button.css [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+.like-filled-icon-btn{
+    color: var(--white);
+    cursor: pointer;
+    background-color: var(--primary-dark-color);
+    padding: 0.5rem;
+    border-radius: 50%;
+    font-size: 1.5rem;
+    margin: 1rem;
+    transition: all 0.5s ease;
+
```
Comment (sakleprajwal) [inline src/CSS/button.css:50]: Avoid blank spaces. You can do the changes in next PR.

**Review by sakleprajwal — APPROVED** [review]
Everything looks good. Please go ahead

File: src/CSS/button.css [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    min-width: 8rem;
+}
+
+.btn-secondary{
+    background-color: var(--secondary-color) !important;
+    font-size: 1.5rem !important;
+}
+
+.btn-outline-primary{
+    border: 2px solid var(--primary-dark-color) !important;
```
Comment (harshkotkar5557) [inline src/CSS/button.css:17]: you should work on its specificity. instead use of important.

**Review by harshkotkar5557 — APPROVED** [review]

File: src/CSS/button.css (hunk 2 above)
Comment (Mahi24-prog) [reply src/CSS/button.css:17]: I use this to overwrite component library defined property

**Review by vaibhav18matere — APPROVED** [review]
looks clean , you can merge this

**Pull request merged**